operations/patch_util.go

```diff
@@ -454,8 +454,16 @@ func (p *patchParams) loadTasks(conf *ClientSettings) error {
 				return errors.Wrap(err, "setting default tasks")
 			}
 		}
+		if len(defaultTasks) > 0 {
+			grip.Info("Using default tasks set in local config")
+		}
 	} else if p.Alias == "" && !p.isUsingLocalAlias {
-		p.Tasks = conf.FindDefaultTasks(p.Project)
+		defaultTasks := conf.FindDefaultTasks(p.Project)
+		p.Tasks = defaultTasks
+		if len(defaultTasks) > 0 {
+			grip.Info("Using default tasks set in local config")
```

Comment:
Should there also be a log when using default variants from the local config as well? Seems like users could be confused by using default variants just like default tasks.

Reply:
added for variants and aliases